frontend/src/utils/index.js

```diff
@@ -484,19 +486,19 @@ export function selfTerminationDaysForSecret (secret) {
 }
 
 export function purposesForSecret (secret) {
-  return selfTerminationDaysForSecret(secret) ? ['evaluation'] : ['evaluation', 'development', 'production']
+  return selfTerminationDaysForSecret(secret) ? ['evaluation'] : ['evaluation', 'development', 'production', 'testing']
```

Comment:
we should add descriptions for the purposes (maybe in a create a separate issue?), as e.g. for `testing` purpose the clusters won't have a monitoring and logging stack deployed

Reply:
done